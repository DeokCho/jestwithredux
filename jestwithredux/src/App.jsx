import LearnJest from "./pages/LearnJest";

const App = () => {
 return (
  <>
   <LearnJest />
  </>
 );
};

export default App;
