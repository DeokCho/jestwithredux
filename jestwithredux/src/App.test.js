import renderer from "react-test-renderer";
import App from "./App";

describe("renders learn react link", () => {
 let component = null;
 it("renders correctly", () => {
  component = renderer.create(<App />);
 });
 it("matches snapshot", () => {
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
 });
});
