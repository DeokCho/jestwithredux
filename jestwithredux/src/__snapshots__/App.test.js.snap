// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link matches snapshot 1`] = `"aaa"`;
